src/Codeception/Subscriber/Console.php

```diff
@@ -576,18 +656,14 @@ public function detectWidth()
         return $this->width;
     }
 
-    private function isWin()
+    private function isWin(): bool
     {
         return strtoupper(substr(PHP_OS, 0, 3)) === 'WIN';
```

Comment:
I had to use this logic in Run command :)

Reply:
if there was a dedicated class for this it would be nice, better than repeating that from `strtoupper(substr(...` in various places.

Reply:
Worse, it is [different](https://github.com/Codeception/Codeception/commit/4d62088152e5459e977a941e0d6a881fe61dd5f9).

[PathResolver](https://github.com/Codeception/Codeception/blob/5.0/src/Codeception/Util/PathResolver.php#L130-L142) has `isWindows` method which only checks DIRECTORY_SEPARATOR and `isPathAbsolute` method which uses regexp pattern for Windows paths.

[Coverage\Subscriber\Printer](https://github.com/Codeception/Codeception/blob/5.0/src/Codeception/Coverage/Subscriber/Printer.php) has its own `absolutePath` method which checks if the first character is `/` or the second character is `:`.
I discovered these methods through your PRs.